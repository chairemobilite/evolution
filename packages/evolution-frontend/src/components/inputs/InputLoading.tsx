/*
 * Copyright 2022, Polytechnique Montreal and contributors
 *
 * This file is licensed under the MIT License.
 * License text available at https://opensource.org/licenses/MIT
 */
import React from 'react';
import { WithTranslation, withTranslation } from 'react-i18next';

export const InputLoading: React.FunctionComponent<WithTranslation> = (props: WithTranslation) => (
    <div className="loader">
        <p>{props.t('main:Loading')}...</p>
        {/*<img className="loader__image" src="/images/loader.gif" />*/}
    </div>
);

export default withTranslation()(InputLoading);
