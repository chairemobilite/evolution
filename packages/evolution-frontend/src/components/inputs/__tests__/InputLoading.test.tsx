/*
 * Copyright 2022, Polytechnique Montreal and contributors
 *
 * This file is licensed under the MIT License.
 * License text available at https://opensource.org/licenses/MIT
 */
import React from 'react';
import TestRenderer from 'react-test-renderer';

import InputLoading from '../InputLoading';

test('Should correctly render Input Loading', () =>{
    const wrapper = TestRenderer.create(<InputLoading />);
    expect(wrapper).toMatchSnapshot();
});
