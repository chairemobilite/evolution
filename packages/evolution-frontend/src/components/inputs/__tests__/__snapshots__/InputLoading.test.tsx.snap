// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should correctly render Input Loading 1`] = `
<div
  className="loader"
>
  <p>
    Loading
    ...
  </p>
</div>
`;
